config/routes.js

```diff
@@ -56,7 +56,7 @@ module.exports.routes = {
   'post /api/v1/pokemon/multi': 'PokemonController.uploadMultipleFiles',
 
   'get /api/v1/pokemon/:id': 'PokemonController.get',
-  'get /api/v1/pokemon/:id/pk6': 'PokemonController.download',
+  'get /api/v1/pokemon/:id/pk': 'PokemonController.download',
```

Comment:
No, but what does "raw" even mean. "pk" tells you you are getting the appropriate "pk" file depending on the generation.

It really doesn't matter anyway, because it's an api call, nothing else. So I'm just going to leave it like this, because I'm lazy.